aws-connect

```diff
@@ -208,6 +224,70 @@ elif [ "${action}" == "tunnel" ]; then
     --parameters "{\"portNumber\":[\"22\"],\"localPortNumber\":[\"${port}\"]}" \
     --region "${aws_region}" \
     --profile "${aws_profile}"
+elif [ "${action}" == "document" ]; then
+
+  if [ -z "${github_token_location}" ]
+  then
+    echo "No github token provided - scripts referenced in SSM document must be in a public repo if no token is provided"
+  else
+    arguments="githubTokenLocation='${github_token_location}',"
+  fi
+
+  if [ -z "${document_parameters}" ]
```

Comment:
general comment.  Some places we use this if syntax, others we use

```
if [ "${action}" == "ssh" ]; then
```

I know I use the single line then format.  I'm not super picky which one we use but keep it consistent within the same script.  multiple styles in the same file make it harder to read.